lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,88 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCodeHash BINARY(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  PRIMARY KEY(uid, unblockCodeHash)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createUnblockCode_1` (
+    IN inUid BINARY(16),
+    IN inCodeHash BINARY(32),
+    IN inCreatedAt BIGINT SIGNED
+)
+BEGIN
+    INSERT INTO unblockCodes(
+        uid,
+        unblockCodeHash,
+        createdAt
+    )
+    VALUES(
+        inUid,
+        inCodeHash,
+        inCreatedAt
+    );
+END;
+
+CREATE PROCEDURE `consumeUnblockCode_1` (
+    inUid BINARY(16),
+    inCodeHash BINARY(32)
+)
+BEGIN
+    DECLARE timestamp BIGINT;
+    SET @timestamp = (
+        SELECT createdAt FROM unblockCodes
+        WHERE
+            uid = inUid
+        AND
+            unblockCodeHash = inCodeHash
+        LIMIT 1
```

Comment:
nit: `LIMIT 1` is redundant here because `PRIMARY KEY(uid, unblockCodeHash)` ensures there can be only one.